src/main/java/me/petterim1/welcomeform/Main.java

```diff
@@ -96,24 +96,20 @@ public void onFormResponse(PlayerFormRespondedEvent e) {
     }
 
     private void showForm(Player p) {
-        FormWindowSimple form = new FormWindowSimple(placeholderFunc.apply(formTitle, p), placeholderFunc.apply(formText, p));
-        if (button) form.addButton(new ElementButton(placeholderFunc.apply(buttonText, p)));
-        p.showFormWindow(form);
+        if (api != null) {
+            formTitle = api.translateString(formTitle, p);
+            formText = api.translateString(formText, p);
+            buttonText = api.translateString(buttonText, p);
+            FormWindowSimple form = new FormWindowSimple(formTitle, formText);
+            if (button) form.addButton(new ElementButton(buttonText));
+            p.showFormWindow(form);
+        }
     }
 
     private void initPlaceholders() {
         try {
-            Class<?> placeholderAPI = Class.forName("com.creeperface.nukkit.placeholderapi.api.PlaceholderAPI");
-            placeholderFunc = (s, p) -> {
-                try {
-                    return (String) placeholderAPI.getDeclaredMethod("translateString", String.class, Player.class).invoke(PlaceholderAPI.getInstance(), s, p);
-                } catch (Exception e) {
-                    getLogger().error("Error with PlaceholderAPI", e);
-                    return s;
-                }
-            };
-        } catch (ClassNotFoundException ignored) {
-            placeholderFunc = (s, p) -> s;
```

Comment:
Please allow the plugin to work without PlaceholderAPI

Reply:
fixed